adafruit_displayio_layout/widgets/icon_widget.py

```diff
@@ -89,3 +166,125 @@ def contains(self, touch_point):  # overrides, then calls Control.contains(x,y)
         touch_y = touch_point[1] - self.y
 
         return super().contains((touch_x, touch_y, 0))
+
+    def selected(self, touch_point):
```

Comment:
With regards to naming: If we use "selected" and "unselected" I think they should be roughly the opposite of each other. So if selected is going to do the zoom in, then I think unselected should do the zoom out.

Truthfully I lean toward wanting some different words.  `selected` is kind of nice since it's in the Button API already. But I think the name can be somewhat confusing because "selected" sometimes mean "has focused" not "being clicked". 

I think we should use names that describe the animation. That way folks can tell what they do at a glance and call the ones they want to use when appropriate.

So for the animations in this IconWidget so far I would recomend: `zoom_animation()`, `zoom_out_animation()`, `colorcycle_animation()`